pages/submissions/[id]/index.tsx

```diff
@@ -0,0 +1,80 @@
+import { GetServerSideProps } from 'next';
+import Head from 'next/head';
+import PersonWidget from '../../../components/PersonWidget/PersonWidget';
+import TaskList from '../../../components/TaskList/TaskList';
+import TaskListHeader from '../../../components/TaskList/TaskListHeader';
+import { Form } from '../../../data/flexibleForms/forms.types';
+import { Resident } from '../../../types';
+import { getProtocol } from 'utils/urls';
+
+interface Props {
+  completedSteps: string[];
+  person: Resident;
+  form: Form;
+}
+
+const TaskListPage = ({
+  completedSteps,
+  person,
+  form,
+}: Props): React.ReactElement => {
+  return (
+    <>
+      <Head>
+        <title>{form?.name} | Social care | Hackney Council</title>
+      </Head>
+      <h1 className="lbh-heading-h1 govuk-!-margin-bottom-8">{form?.name}</h1>
+      <div className="govuk-grid-row">
+        <div className="govuk-grid-column-two-thirds">
+          <TaskListHeader
+            steps={form?.steps}
+            completedSteps={completedSteps}
+            onFinish={() => {
+              null;
+            }}
+          />
```

Comment:
We can remove the form null conditional operators.

form.? shouldn't be required as form is not optional.